main/main.cpp

```diff
@@ -903,7 +903,10 @@ Error Main::setup2() {
 
 	path_remap->load_remaps();
 
-	if (show_logo) { //boot logo!
+	List<String> args = OS::get_singleton()->get_cmdline_args();
+	bool editor = args.size()>0 && (args.find("-e")>=0 || args.find("-editor")>=0);
```

Comment:
I found a bug that godot logo doesn't show up when editor starts.
I'm facing problem with squashing commits now again :(
And messed up again.
I will make a change as you suggest and check your test case
then will make another PR for this.